bigbluebutton-html5/imports/api/presentations/server/methods/publishSwitchToXSlideMessage.js

```diff
@@ -6,32 +6,35 @@ import Slides from '/imports/api/slides';
 import { redisConfig } from '/config';
 
 Meteor.methods({
-  publishSwitchToNextSlideMessage(credentials) {
+  publishSwitchToXSlideMessage(credentials, requestedSlideNum) {
```

Comment:
I a better name would be SwitchToSpecificSlideMessage or just SwitchSlideMessage.